.travis.yml

```diff
@@ -10,5 +10,5 @@ before_install:
 script:
   - go test -v ./...
 after_success:
-  - gox -output "dist/{{.OS}}_{{.Arch}}_{{.Dir}}"
-  - ghr --username iangrunert --token $GITHUB_TOKEN --replace --prerelease --debug pre-release dist/
\ No newline at end of file
+  - scripts/build.sh
```

Comment:
Ah! Right you are, I didn't notice that in the Travis UI. There's just a slight change that needs to happen on the -output format to get things working then we should be able to merge this.